pkg/manifests/updates.go

```diff
@@ -154,6 +184,105 @@ func UpdateResourceTopologyExporterCommand(args []string, vars map[string]string
 	return res
 }
 
+func UpdateMachineConfig(mc *machineconfigv1.MachineConfig, name string, mcpSelector *metav1.LabelSelector) error {
+	if name != "" {
+		mc.Name = fmt.Sprintf("51-%s", name)
+	}
+
+	ignitionConfig, err := getIgnitionConfig()
```

Comment:
ok, this is why the error surfaces up to `Update()`. We should load all the assets we need explicitely. Moving this code into `GetManifests` is still a stretch (selinux policies and ignition config don't really quality as `manifests`) but since they are required for the MachineConfig **manifest** they still kinda fit into the picture.

This cleanup can be done later, though.